chia/server/ws_connection.py

```diff
@@ -558,3 +558,6 @@ def get_peer_logging(self) -> PeerInfo:
             return PeerInfo(self.peer_host, port)
         else:
             return info
+
+    def has_capability(self, Capability) -> bool:
+        return Capability in self.peer_capabilities
```

Comment:
https://github.com/Chia-Network/chia-blockchain/actions/runs/3592717699/jobs/6048798710#step:9:100
> `chia/server/ws_connection.py:562: error: Function is missing a type annotation for one or more arguments  [no-untyped-def]`


```suggestion
    def has_capability(self, capability: Capability) -> bool:
        return capability in self.peer_capabilities
```

Reply:
damn, thanks